data/fwupd.service.in

```diff
@@ -13,6 +13,16 @@ RuntimeDirectory=@motd_dir@
 RuntimeDirectoryPreserve=yes
 BusName=org.freedesktop.fwupd
 ExecStart=@libexecdir@/fwupd/fwupd
+
+# Already implicitly dropped, dropping explicitly
```

Comment:
what's the logic to duplicate?

Reply:
Just to be aware of them if whitelisting point is reached, no other purpose. More a note that these have essentially been tested already than anything else.